lib/inferno/dsl/fhir_evaluation/evaluator.rb

```diff
@@ -19,6 +19,10 @@ def initialize(ig) # rubocop:disable Naming/MethodParameterName
         def evaluate(data, config = Config.new)
           context = EvaluationContext.new(@ig, data, config)
 
+          Dir.glob(File.join(__dir__, 'rules', '*.rb')).each do |file|
```

Comment:
If I recall correctly, one of Steve's comments on a previous PR was that this should be at the top of the file with the rest of the requires, not in the evaluate function. (Do you still have access to those earlier comments?)

Reply:
I believe the comment was this: https://github.com/inferno-framework/inferno-core/pull/557#discussion_r1866305682

Reply:
Moved these at top of the file same as other requires